### PR TITLE
Movie general search title as link

### PR DESCRIPTION
**Pull request by nicorithner**
### Description:

Added missing elements from story 'Movies Page'.
The general movies search bar takes a keyword such as 'Phoenix' for example and returns movie titles that include that word.
The movie title are links. In the next story we will make them lead to the movie show page

### Tasks:

- [x] Upgrade search form and test
- [x] Added link to titles and created test
- [x] Rubocop fixes

#### Notes:
 Title links are blank right but they will be upgraded in the next story

### CODE REVIEW

File: app/views/movies/index.html.erb [hunk 1]
```diff
@@ -6,19 +6,16 @@
...
 </div>
 
 <div class="results">
   <table>
-    <% @top_40.each do |movie| %>
+    <% @movies.each do |movie| %>
       <div class="movie">
         <tr>
-          <td class='title'><%= movie[:title] %></td>
+          <td class='title'><%= link_to movie[:title], "#" %></td>
```
Comment (nicorithner) [inline app/views/movies/index.html.erb:18]: Title links are blank right but they will be upgraded in the next story

**Review by nicorithner — COMMENTED** [review]
There are more Rubocop fixes needed in movies_controller but we are leaving them for when we refactor

File: app/controllers/movies_controller.rb [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-      body_2 = JSON.parse(response_2.body, symbolize_names: true)
-      @top_40 = (body_1[:results] << body_2[:results]).flatten
+      response1 = Faraday.get "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV['TMDB_API_KEY']}&page=1"
+      response2 = Faraday.get "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV['TMDB_API_KEY']}&page=2"
+      body1 = JSON.parse(response1.body, symbolize_names: true)
+      body2 = JSON.parse(response2.body, symbolize_names: true)
+      @movies = (body1[:results] << body2[:results]).flatten
     end
   end
 end
```
Comment (hashmaster3k) [inline app/controllers/movies_controller.rb:17]: RuboCop threw concerns for method and string lengths being too long. This controller will be refactored soon.

File: app/views/discover/index.html.erb [hunk 3]
```diff
@@ -4,9 +4,6 @@
 
   <h3>OR</h3>
   <div class="search_movie_form">
-    <%= form_with url: movies_path, method: :get, local: true do |form| %>
-      <%= form.text_field :q, placeholder: "Search movie by title" %>
-      <%= form.submit "Find Movies" %>
-    <% end %>
+    <%= render partial: './search_form' %>
```
Comment (hashmaster3k) [inline app/views/discover/index.html.erb:7]: Great use of partials

**Review by hashmaster3k — APPROVED** [review]

**Pull request merged**